smarty/templates/instrumentstatus_controlpanel.tpl

```diff
@@ -1,67 +1,84 @@
 <script type="text/javascript" src="{$baseurl}/js/instrument_controlpanel_control.js"></script>
-
+<script type="text/javascript" src="{$baseurl}/htdocs/vendor/sweetalert/sweetalert.js"></script>
```

Comment:
This line for including the sweetalert.js may be unnecessary.

Reply:
deleted the line (was giving an error)